src/modules/membership.js

```diff
@@ -20,7 +20,7 @@ class Membership extends Service {
     }
     let synchronized = '';
     if (opt.sync !== undefined && opt.sync === true) {
-      synchronized = '?synchronized=true';
+      synchronized = '?synchronized';
```

Comment:
If setting synchronized=true is the same, let's revert. It's more clear what is happening.

Reply:
Due to feedback from Jim at GWS, we'll use synchonized. Please add comment at all calls about the reason for the odd use.